code/modules/clothing/under/striketeam.dm

```diff
@@ -0,0 +1,116 @@
+var/list/deathsquad_uniforms = list()
+
+/obj/item/clothing/under/deathsquad
+	name = "deathsquad holosuit"
+	desc = "A state-of-the-art suit featuring an holographic map of the area, to help the squad coordinate their efforts."
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/altsuits.dmi', "right_hand" = 'icons/mob/in-hand/right/altsuits.dmi')
+	icon_state = "deathsquad"
+	item_state = "deathsquad"
+	_color = "deathsquad"
+	flags = FPRINT  | ONESIZEFITSALL
+	action_button_name = "Toggle Holomap"
+	var/mob/living/carbon/human/activator = null
+	var/holomap_activated = 0
+	var/list/holomap_images = list()
+
+/obj/item/clothing/under/deathsquad/New()
+	..()
+	deathsquad_uniforms += src
+
+/obj/item/clothing/under/deathsquad/Destroy()
+	deathsquad_uniforms -= src
+	..()
+
+/obj/item/clothing/under/deathsquad/verb/togglemap()
+	set name = "Toggle Holomap"
+	set category = "Object"
+	set src in usr
+	if(usr.isUnconscious())
+		return
+
+	if(!ishuman(usr))
+		to_chat(usr, "<span class='warning'>Only humanoids can wear this suit</span>")
+		return
+
+	var/mob/living/carbon/human/H = usr
+
+	if(H.get_item_by_slot(slot_w_uniform) != src)
+		to_chat(usr, "<span class='warning'>You need to wear the suit first</span>")
+		return
+
+	if(src.holomap_activated)
+		holomap_activated = 0
+		activator.client.images -= holomap_images
+		activator = null
+		holomap_images.len = 0
+		processing_objects.Remove(src)
+		to_chat(usr, "<span class='notice'>You disable the holomap.</span>")
+	else
+		holomap_activated = 1
+		activator = usr
+		processing_objects.Add(src)
+		process()
+		to_chat(usr, "<span class='notice'>You enable the holomap.</span>")
+
+/obj/item/clothing/under/deathsquad/process()
+	var/turf/T = get_turf(src)
+	if(!T)//nullspace begone!
+		return
+
+	if((activator.get_item_by_slot(slot_w_uniform) != src) || (!activator.client) || (holominimaps[T.z] == null))
+		activator = null
+		holomap_activated = 0
+		processing_objects.Remove(src)
+		return
+
+	activator.client.images -= holomap_images
+
+	holomap_images.len = 0
+
+	var/image/bgmap
+
+	if(T.z == map.zCentcomm)
+		bgmap = image(centcommminimaps["deathsquad"])
+	else
+		bgmap = image(holominimaps[T.z])
+	bgmap.pixel_x = -1*T.x + 240
+	bgmap.pixel_y = -1*T.y + 241
+	bgmap.plane = HUD_PLANE
+	bgmap.layer = HUD_BASE_LAYER
+	bgmap.color = "#0B74B4"
+	bgmap.loc = activator.hud_used.holomap_obj
+	holomap_images += bgmap
+
+	for(var/obj/item/clothing/under/deathsquad/D in deathsquad_uniforms)
```

Comment:
Then you can have a suit store the images for the others and modify pixel_x as necessary without creating a new image every process().